### PR TITLE
Set correct default value for tag_text on headlines

### PR DESCRIPTION
**Pull request by dgwatkins**
Closes https://onthegosystems.myjetbrains.com/youtrack/issue/wpmltm-4108

### CODE REVIEW

**Comment by dgwatkins** [comment]
Please review @OnTheGoSystems/compatibility 

Please note that this code is covered with CC tests.

File: src/class-wpml-cornerstone-translatable-nodes.php [hunk 1]
```diff
@@ -144,8 +144,8 @@ public function get_string_name( $node_id, $field, $settings ) {
 	 * @return string
 	 */
 	private function get_wrap_tag( $settings ) {
-		if ( isset( $settings['_type'] ) && 'headline' === $settings['_type'] && isset( $settings['text_tag'] ) ) {
-			return $settings['text_tag'];
+		if ( isset( $settings['_type'] ) && 'headline' === $settings['_type'] ) {
+			return isset( $settings['text_tag'] ) ? $settings['text_tag'] : 'h1';
```
Comment (strategio) [inline src/class-wpml-cornerstone-translatable-nodes.php]: You can also use this @dgwatkins:

```php
return Obj::propOr( 'h1', 'text_tag', $settings );
```

**Review by strategio — DISMISSED** [review]
Looks good @dgwatkins.

This reminds me we have to merge these packages into the new WPML Page Builders addon...

File: src/class-wpml-cornerstone-translatable-nodes.php (hunk 1 above)
Comment (dgwatkins) [reply src/class-wpml-cornerstone-translatable-nodes.php]: Thanks, applied suggestion.

**Comment by dgwatkins** [comment]
The CC tests are running local. I will run them in the CI as soon as this is merged.

@OnTheGoSystems/compatibility second review please?

**Review by strategio — DISMISSED** [review]

**Pull request merged**